app.js

```diff
@@ -1,8 +1,84 @@
-// import functions
+import { getPokedex } from './fetch.js';
 
+// import functions
+console.log('Its ALIVE!');
 // grab DOM elements
+const template = document.querySelector('#template');
+const list = document.querySelector('#list');
+const audio = document.querySelector('#audio');
+const response = document.querySelector('#response');
+const input = document.querySelector('#Answer');
+const button = document.querySelector('#button');
 
+console.log('test', template, list, audio, response, input);
 // set event listeners 
     // get user input
     // use user input to update state 
     // update DOM to reflect the new state
+
+input.addEventListener('submit', () => {
+
+    const inputValue = input.value;
+    console.log(inputValue);
+    
+    if ('Answer' === 'Yes') {
+
+        response.textContent = 'You have good taste.';
+
+    } 
+    else {
+
+        response.textContent = 'Please answer correctly.';
+    
+    }
+
+});
+
+button.addEventListener('click', () => {
+
+ 
+    let answer = input.value;
+    if (answer === 'Yes' || answer === 'yes') {
```

Comment:
Good thinking with watching for the case sensitivity! A handy way to resolve this is to transform the value to all UPPERCASE or all lowercase: 

`answer.toLowerCase()` or `answer.toUpperCase()`

Ref: 
- [MDN - String.prototype.toLowerCase()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase)
- [MDN - String.prototype.toUpperCase()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toUpperCase)